File: UNIT 5/Assesment/456.c
#include <stdio.h>
#include <math.h>

float cariAkar(float a, float b, float c) {
    float D = (b*b) - (4*a*c);
    if (D < 0) {
        return -9999; 
    }
    float akar1 = (-b + sqrt(D)) / (2*a);
    return akar1;
}
float cariAkar(float a, float b, float c);
#include <stdio.h>

int main() {
    float a, b, c;
    printf("Masukkan nilai a, b, c: ");
    scanf("%f %f %f", &a, &b, &c);
    
    float hasil = cariAkar(a, b, c);
    printf("Akar persamaan: %.2f", hasil);
    return 0;
}

File: UNIT 5/Assesment/456.py
import math

def cariAkar(a, b, c):
    D = (b * b) - (4 * a * c)
    if D < 0:
        return None  # menandakan akar tidak real
    akar1 = (-b + math.sqrt(D)) / (2 * a)
    return akar1
# Prototype konsep:
# def cariAkar(a: float, b: float, c: float) -> float:
def main():
    a = float(input("Masukkan nilai a: "))
    b = float(input("Masukkan nilai b: "))
    c = float(input("Masukkan nilai c: "))

    hasil = cariAkar(a, b, c)

    if hasil is None:
        print("Akar tidak real")
    else:
        print(f"Akar persamaan adalah: {hasil:.2f}")

main()
